### ISSUE TITLE
owncloud client 2.1.1: accounts button not visible

### ISSUE THREAD

**Issue by Hollywood777** [owncloud/client#4980]
### Expected behaviour
1st issue: OwnCloud Client should show Account Button in ANY case
2nd issue: de-installation with purge should remove all config data

### Actual behaviour
Account Button is not shown, if account is set up, but no sync directory is activated in installation assistant
De-installation with apt-get purge is not cleaning all configuration data; assistant is not invoked after
re-installation anymore

### Steps to reproduce
1. installation of OwnCloud client 2.1.1 via Synaptic under Xubuntu 16.04
2. Following the Assistant, entering Server Data
3. De-selection of all found and offered sync directories, (since this should be set up later)
4. problem: button "Account" is not visible in client anymore (even after client re-start/re-install

Tried Solution strategy:
5. De-installation of owncloud client:
    sudo apt-get purge owncloud-client
    sudo apt-get autoremove
    sudo apt-get autoclean
6. removal of owncloud config files under /home/user/.local/share/data/ownCloud
7. Re-installation: sudo apt-get install owncloud-client
    --> still the same problem: configuration assistant is not invoked anymore, Account-Button not 
    visible, no sync-directory can be configured

Therefore I see the following three issues to be fixed:
1. The Accout Button should be visible in any case (in my case the account has been entered, but
    just no sync directory activated ---> in this case additionally the entered account should be visible
    (much better: program should have a pull-down menu which works in any case)
2. In the documentation information is missing how to manually clean up the client data and
    where the account settings are stored (obviously not in /home/user/.local/share/data/ownCloud
    and not in /etc/...)
    Therefore a detailed documentation in the client user manual is need, where which configuration files
    are stored and where the account information is stored and how it can be deleted manually.
3. The un-installer is not cleaning up all configuration data, once the 'purge' option is applied in combination with apt-get, this should be definitively fixed

### Server configuration
Operating system: Raspian

Web server:  nginx

Database: mysql

PHP version: 5.6.20

ownCloud version: 8.2.1

Storage backend: local disk

### Client configuration
Client version: 2.1.1+dfsg-1ubuntu1.1

Operating system: Xubuntu 16.04 

OS language: german

Installation path of client: standard path

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

```Template for output < 10 lines```

1. Output of `owncloud --logwindow` or `owncloud --logfile log.txt`
(On Windows using `cmd.exe`, you might need to first `cd` into the ownCloud directory)
(See also http://doc.owncloud.org/desktop/2.2/troubleshooting.html#client-logfile )

2. Web server error log: n.a.

3. ownCloud log (data/owncloud.log): no logs generated

**Comment by guruz** [comment]
You should see the configuration wizard pop up some seconds after deleting the last account.

Are you sure this is not happening?
Maybe the window is on another virtual desktop or screen?

**Comment by Hollywood777** [comment]
I didn't delete the account.
When I opened the client for the first time, I entered the server data, connected to the server and got a list of all directories, which are candidates for synchronisation. I un-checked ALL of them (since I wanted to configure that later on) and then the client didn't show me the account (the login data I had entered) and also not the "Account" button.

I connected the Server manually: worked, then I deleted all Owncloud client config files I could find, rebooted - didn't work.

The interesting thing: I just started Xubuntu again to write you the name of the three buttons I see and now (3 days later) it works!
Could it be that some other packages were updated in the last two days,
which have an impact?
